packages/react/src/components/Search/Search.js

```diff
@@ -53,7 +53,7 @@ export default class Search extends Component {
     /**
      * Specify a label to be read by screen readers on the "close" button
      */
-    closeButtonLabelText: PropTypes.string,
+    closeButtonLabelText: PropTypes.string.isRequired,
```

Comment:
A question; Does this work with "falling back to `defaultProps`" scenario?

Reply:
I removed the requirement on this prop